### DISCUSSION TITLE
Organizationへの参加するための管理者への連絡について

### DISCUSSION THREAD

**Discussion by yukyt** [increments/qiita-discussions#590] [ヘルプ] [answered]
https://help.qiita.com/ja/articles/qiita-org-member
こちらのページでOrganizationへの参加方法について記載されており、管理者からしか招待できないというのは理解したのですが、
Organizationのメンバー一覧では誰が管理者なのか分かりませんでした。

そのためどうアプローチしたものか悩んでいます。
メンバーには管理者が誰か見えているのならば、メンバーをアカウント名から何とか特定し、管理者が誰か聞くとか、
あるいはURLなどから代表の連絡先に連絡をするというのが思いつきますが、
もう少しスマートな連絡方法は無いでしょうか？

**Comment by tanaka-asami415** [comment, marked as answer]
@yukyt 
お問い合わせありがとうございます🙏

現在、メンバーからも管理者は分からない仕様となっております。

Organizationで、外部公開用メールアドレスが設定されていれば
概要ページの「基本情報」に連絡用メールアドレスが表示されておりますので、
記載のメールアドレスよりご連絡していただくことが可能です🙇
https://qiita.com/organizations/qiita-inc/profile

**Comment by yukyt** [comment]
ご回答ありがとうございます。承知いたしましたので本件はクローズいたします。

また、今後同様に迷う人がでないよう、要望として以下をご検討いただけますと幸いです。
・[Organizationの参加・脱退方法](https://help.qiita.com/ja/articles/qiita-org-member)にご回答いただいた内容を追記する。
・Organizationに加入申請のリンクを追加する。
　可能であればオプションを設けた上で、オンの場合のみ表示し、オフの場合は申請は受け付けていないことを加入希望者に明示。

**Reply by tanaka-asami415** [reply]
@yukyt 
ありがとうございます🙏
ヘルプへの追記を行い、申請のリンク追加に関しては今後の改善の参考にさせていただきます！
